<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formulaireCycle</class>
 <widget class="QWidget" name="formulaireCycle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>formulaireCycle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Jour deL ivraison</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Jour du d√©but du cycle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Jour d'ouverture des ventes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Jour de fin de vente</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dateDC"/>
     </item>
     <item row="1" column="2">
      <widget class="QDateEdit" name="dateDV"/>
     </item>
     <item row="2" column="2">
      <widget class="QDateEdit" name="dateFV"/>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="dateL"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ajouter Cycle de Vente</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
